deploy.sh

```diff
@@ -549,7 +562,7 @@ do
     echo "Binder IP: ${BINDER_IP}" | tee binder-ip.log
 done
 
-if [[ -n $BINDERHUB_CONTAINER_MODE ]] ; then
+if [[ -n $BINDERHUB_CONTAINER_MODE ]] || [[ "${LOG_TO_BLOB_STORAGE,,}" = 'true' ]] ; then
```

Comment:
What's the purpose of the two commas in this line?

Reply:
@sgibson91 I am also very much a newbie when it comes to bash scripting, so no worries there.

That said, from my understanding, it will lower-case the `LOG_TO_BLOB_STORAGE` value; since it's user input, I wanted to reduce friction, as per:

https://stackoverflow.com/a/19411918/50776

However, I realize now that it wouldn't be valid JSON if it _wasn't_ lower case.

I'll make it a case-sensitive comparison today.

Reply:
Apologies for the delay.

I've addressed this in the latest push.

LMK if there's anything else that I can do.

Reply:
Thanks so much for your hard work!